tests/cypress/support/helpers/tenderly/account.ts

```diff
@@ -0,0 +1,12 @@
+export type TenderlyAccount = {
+  accountSlug: string
+  projectSlug: string
+  accessKey: string
+}
+
+const accountSlug = Cypress.env('TENDERLY_ACCOUNT_SLUG')
+const projectSlug = Cypress.env('TENDERLY_PROJECT_SLUG')
+const accessKey = Cypress.env('TENDERLY_ACCESS_KEY')
+
+export const tenderlyAccount =
```

Comment:
tbh I'd rather throw an error if you import this file without the proper environment set.
If needed move it to a function. It makes much easier to find problems in your local/ci setup.

Reply:
Sure. For the time being the tests will fail on Github Actions until we get an account set up on the Github repository